.gitignore

```diff
@@ -17,3 +17,10 @@ yarn-debug.log*
 .yarn-integrity
 yarn-error.log
 
+
+/public/packs
+/public/packs-test
+/node_modules
+/yarn-error.log
+yarn-debug.log*
+.yarn-integrity
```

Comment:
**suggestion / blocking**: I think some duplicate lines snuck in. Would be good to de-dupe before merging.